tests/integration_tests/build/test_coverage.py

```diff
@@ -20,7 +20,7 @@
 
 
 COVERAGE_TARGET_PCT = 81.0
-# TODO: Put the coverage in s3 and update it automatically.
+COVERAGE_MAX_DELTA = 0.1
```

Comment:
I feel this PR warrants our first ever meme code comment.

![why not 0.01%?](https://i.imgflip.com/2scntz.jpg)

```suggestion
COVERAGE_MAX_DELTA = 0.01
```

But really, let's work backwards from how many untested lines we want to allow ... 0.01% results in like 2-3, which might be enough.

Reply:
I am for 0.01 delta (well actually I am for 0.00000001 delta, but I am a woman that can make compromises).